weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/TraceUsageStats.tsx

```diff
@@ -13,18 +13,13 @@ import React, {ReactNode} from 'react';
 import {MOON_600} from '../../../../../../common/css/color.styles';
```

Comment:
This file will slowly be deleted after i update cost stats in the trace view

Reply:
So the trace view is still using client-side cost calcs, correct?

Reply:
yes